home/ivp-planning/user-interface/the-3d-panel.md

```diff
@@ -1,28 +1,30 @@
-| ![](../../../.gitbook/assets/iVP\_icon\_3D\_update\_selected\_camera.jpg)          |                                                                                                                                                                                              |
+|                                                               |                                                                                                                                                   |
+| ------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ![](../../../.gitbook/assets/3D_panel/Icon_camera_mode.jpg)   | Toggle camera mode between free camera mode and bound to                                                                                          |
+| ![](../../../.gitbook/assets/3D_panel/Icon_cameralist.jpg)    | Allows to switch between different [virtual cameras](../advanced-tools/virtual-cameras.md)                                                        |
+| ![](../../../.gitbook/assets/3D_panel/Icon_aspect.jpg)        | Allows to switch between different aspect ratios for the layout                                                                                   |
+| ![](../../../.gitbook/assets/3D_panel/Icon_move.jpg)          | Switches to a mode that makes it possible to [move ](../machines/selecting-and-moving-objects.md)the selected object(s) by using the mouse        |
+| ![](../../../.gitbook/assets/3D_panel/Icon_rotate.jpg)        | Switches to a mode that makes it possible to [rotate ](../machines/scale-and-rotate-objects.md#rotate-objects)the selected object(s)              |
+| ![](../../../.gitbook/assets/3D_panel/Icon_scale.jpg)         | Switches to a mode that makes it possible to [scale ](../machines/scale-objects.md#scaling-objects-in-the-3d-panel)the selected object(s)         |
+| ![](../../../.gitbook/assets/3D_panel/Icon_screenshot.jpg)    | Creates a [screenshot ](../advanced-tools/creating-screenshots.md) of the current camera view in the [3D view](../user-interface/the-3d-panel.md) |
+| ![](../../../.gitbook/assets/3D_panel/Icon_FOV.jpg)           | Changes the field of view of the currently selected [virtual cameras](../advanced-tools/virtual-cameras.md)                                       |
+| ![](../../../.gitbook/assets/3D_panel/Icon_reset_camera.jpg)  | Resets the camera view to a centered default perspective                                                                                          |
+| ![](../../../.gitbook/assets/3D_panel/Icon_add_camera.jpg)    | Adds a new [virtual camera](../advanced-tools/virtual-cameras.md) (Hotkey B)                                                                      |
```

Comment:
Remove the mention of the shortcut. Things like this only make it hard for us to track down all references to the shortcut in case we decide to change it at some time. We already have the shortcut on the Keyboard Shortcuts page and we now also display keyboard shortcuts in tooltips inside Planning 🙂